### PR TITLE
 Issue 554: added links to stage environments for launcher and docs to Contrib Guide

### LINKED ISSUE #554: Include staging URLs in the Contrib Guide
The staging URLs should be included in the Contrib Guide:

* Doc stage: https://appdev.prod-preview.openshift.io/
* Launcher stage: https://launch.prod-preview.openshift.io/

### PR DESCRIPTION
**Pull request by inoxx03**
resolves #554

also created the corresponding attributes in `document-attributes.adoc`

### CODE REVIEW

**Comment by inoxx03** [comment]
@tradej We've already had a link to the docs stage environment in the contrib guide, so i just tweaked it by replacing the URL with an attribute
For the launcher stage, I added a line with a link in the as-of-yet-unwritten section on developing the launcher. WDYT?
Would you prefer if we explicitly spelled out the URLs somewhere or will links do?

File: docs/topics/templates/document-attributes.adoc [hunk 1]
```diff
@@ -144,3 +144,7 @@
 
 :docs-name: appdev.openshift.io
 :link-docs: https://appdev.openshift.io
+
+// stage environment URLs for Docs and Launcher
+:docs-stage: https://appdev.prod-preview.openshift.io/
+:launcher-stage: https://launch.prod-preview.openshift.io/
```
Comment (tradej) [inline docs/topics/templates/document-attributes.adoc]: I'd advocate for using `link-docs-stage` and `link-launcher-stage` to keep consistency with other attributes (and to sort-of create namespaces for the attributes), but that's not a hard requirement, and I'm willing to accept this if necessary.

File: docs/contrib-guide/master.adoc [hunk 2]
```diff
@@ -9,6 +9,8 @@ include::topics/templates/document-attributes.adoc[]
 
 _TBD_
 
+You can preview the latest state of {launcher} by navigating to the  link:{launcher-stage}[stage build] of {launcher}.
```
Comment (rhoads-zach) [inline docs/contrib-guide/master.adoc]: double space between `the` and `link:`

File: docs/topics/templates/document-attributes.adoc (hunk 1 above)
Comment (inoxx03) [reply docs/topics/templates/document-attributes.adoc]: @tradej Makes a lot of sense, I'll change the attribute name:)

File: docs/contrib-guide/master.adoc (hunk 2 above)
Comment (inoxx03) [reply docs/contrib-guide/master.adoc]: @rhoads-zach thanks for catching this, will fix:)

**Comment by inoxx03** [comment]
@rhoads-zach @tradej fixed, please let me know what you think.

**Comment by rhoads-zach** [comment]
LGTM

@tradej do you want to merge, or do you want me to?

**Comment by rhoads-zach** [comment]
merged: 392c32d3a87ee75c4d2c6bd48ef303ea137e5ea4

**Pull request closed without merging**